README.md

```diff
@@ -27,6 +27,14 @@ To use your own config file, set the `LOGSTASH_CONFIG_URL` environment variable
       -p 9200:9200 \
       pblittle/docker-logstash
 
+To use a config file from the local file system, mount the file as a volume using the `-v` option. Replace `/my/logstash.conf` with the absolute path to your config file.
+
+    $ docker run -d \
+      -v /my/logstash.conf:/opt/logstash.conf \
```

Comment:
@remmelt, I have moved the config location to `/opt/logstash/conf.d`. This change removes the very bad `/opt/logstash.conf` restriction. Now any file in `/opt/logstash/conf.d` will get loaded by logstash.